src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocStyleCheck.java

```diff
@@ -327,24 +327,24 @@
     public static final String MSG_EXTRA_HTML = "javadoc.extraHtml";
 
     /** HTML tags that do not require a close tag. */
-    private static final Set<String> SINGLE_TAGS = Collections.unmodifiableSortedSet(
-        Arrays.stream(new String[] {"br", "li", "dt", "dd", "hr", "img", "p", "td", "tr", "th", })
-            .collect(Collectors.toCollection(TreeSet::new)));
+    private static final Set<String> SINGLE_TAGS = new TreeSet<>(Set.of(
```

Comment:
This will create an ordinal (mutable) `TreeSet` and simply call `addAll` method with the inner set items.

Reply:
Thanks for pointing this.
Actually, the only usage of this set is to invoke `contains` method, so there is no need for TreeSet. Changed to usual `Set.of`